clusterloader2/testing/load/config.yaml

```diff
@@ -315,7 +333,7 @@ steps:
     path: /modules/configmaps-secrets.yaml
     params:
       actionName: delete
-      tuningSet: Global100qps
+      tuningSet: default
```

Comment:
Doesn't it simplify test for 100 nodes?

Reply:
As discussed offline, it's true but the point of this part of the test isn't to stress any specific dimension or kube-apiserver, rather to mitigate https://github.com/kubernetes/kubernetes/issues/96635